src/ooxml/java/org/apache/poi/xddf/usermodel/XDDFLineProperties.java

```diff
@@ -293,21 +304,21 @@ public void setTailEnd(XDDFLineEndProperties properties) {
         }
     }
 
-    public Integer getWidth() {
+    public Double getWidth() {
```

Comment:
this is an API change - we probably need an ugly new getWidthAsDouble() and to deprecate the integer version